doc/source/how-to/dns-configuration.rst

```diff
@@ -0,0 +1,93 @@
+DNS configuration
+=================
+
+As explained in :ref:`Documenting`, PyAnsys projects publish their documentation
+online under the following canonical name (CNAME) convention:
+
+``https://<product>.docs.pyansys.com``
+
+In order to request a CNAME for the ``pyansys.com`` domain, please contact
+`Maxime Rey`_, `Roberto Pastor Muela`_ or `Alex Kaszynski`_. Any of these members
+can handle the creation of the requested PyAnsys subdomain.
+
+Once the CNAME is created, repository administrators can configure their published
+documentation in GitHub pages to be exposed through it. In order to configure the CNAME
+for your documentation, please refer to GitHub's online documentation:
+`Managing a custom domain for your GitHub Pages site`_.
+
+PyAnsys verified domains
+------------------------
+
+In the `PyAnsys organization`_, the following domains have been verified:
```

Comment:
Suggesting to use `PyAnsys GitHub organization` here, as this term appears in other parts of the guide:

```suggestion
In the `PyAnsys GitHub organization`_, the following domains have been verified:
```